Sources/Paywalls/Components/PaywallPurchaseButtonComponent.swift

```diff
@@ -24,23 +25,134 @@ public extension PaywallComponent {
             case webProductSelection = "web_product_selection"
         }
 
+        public enum Method: Codable, Sendable, Hashable, Equatable {
```

Comment:
This is, I think, the perfect place for `@_spi`. This way, it can be accessed from RevenueCatUI (which is needed), but it's not really a public API. That way, in the future, if we need to replace it (the same way `Method` replaces `Action`), we could just remove it instead of deprecating it. Not a big deal though, as every other API here is `public` already.